graph.go

```diff
@@ -184,6 +190,12 @@ func (g *Graph) Edge(fromNode, toNode Node, labels ...string) Edge {
 		to:            toNode,
 		AttributesMap: AttributesMap{attributes: map[string]interface{}{}},
 		graph:         edgeOwner}
+	if fromNodePort != "" {
+		e.fromPort = ":" + fromNodePort
```

Comment:
The fromPort field is set to an easy to print value. I would avoid that and only add the ":" when needed on printing on the writer. Also, I an edge will have a getter later, it can return the value as passed on the creation of the edge.